clickhouse-repos-manager/repos.py

```diff
@@ -59,18 +63,26 @@ def __init__(
         self.packages = packages
         self._repo_config = repo_config
         self._repo_root = repo_root
+        self._configs_root = repo_root
         self.logger = logger
         self.check_dirs()
 
     def add_packages(self, version_type: str, *additional_version_types: str) -> None:
-        deb_files = " ".join(pkg.path.as_posix() for pkg in self.packages)
-        command = f"{self.reprepro_cmd} includedeb '{version_type}' {deb_files}"
-        self.logger.info("Deploying DEB packages to codename %s", version_type)
-        self.logger.info(
-            "Deployment logs:\n%s", runner(command, stderr=subprocess.STDOUT)
-        )
-        for additional_version_type in additional_version_types:
-            self.process_additional_packages(version_type, additional_version_type)
+        with self.local_configs():
+            deb_files = " ".join(pkg.path.as_posix() for pkg in self.packages)
+            command = f"{self.reprepro_cmd} includedeb '{version_type}' {deb_files}"
+            self.logger.info("Deploying DEB packages to codename %s", version_type)
+            self.logger.info(
+                "Deployment logs:\n%s", runner(command, stderr=subprocess.STDOUT)
+            )
+            for additional_version_type in additional_version_types:
+                self.process_additional_packages(version_type, additional_version_type)
+            self.logger.info("Sleep for a while to get things synced")
+            runner("sync ; sleep 10")
```

Comment:
It tries to avoid https://github.com/yandex-cloud/geesefs/issues/98#issuecomment-2168457365